lib/code_fund/schemas/user.ex

```diff
@@ -10,6 +10,7 @@ defmodule CodeFund.Schema.User do
     has_many(:campaigns, CodeFund.Schema.Campaign)
     has_many(:properties, CodeFund.Schema.Property)
     has_many(:creatives, CodeFund.Schema.Creative)
+    has_many(:budgeted_campaigns, CodeFund.Schema.BudgetedCampaign)
```

Comment:
Are we planning on querying the user's budgeted campaigns?

Reply:
Yes